playbooks/ansible/roles/sit.xfs/tasks/new_volume.yml

```diff
@@ -2,7 +2,7 @@
 - name: Create the XFS filesystem
   filesystem:
     dev: "{{ volume.xfs.device }}"
-    force: true
+    force: false
```

Comment:
`false` is the default in case you don't want to force the creation of filesystem.

Reply:
I've seen defaults change with ansible. While this setting may not change, I see no harm in explicitly setting it to false.